build-tools/libzip/libzip.mdproj

```diff
@@ -28,5 +29,5 @@
       <ReferenceOutputAssembly>False</ReferenceOutputAssembly>
     </ProjectReference>
   </ItemGroup>
-  <Target Name="Build" DependsOnTargets="$(BuildDependsOn)" />
+  <Target Name="Build" DependsOnTargets="$(BuildDependsOn)" />

```

Comment:
This can be removed now, as `Microsoft.Common.targets` provides a `Build` target.